src/index.ts

```
@@ -27,12 +28,37 @@ const config = {
 	ENVIRONMENT: getInput("environment"),
 	VARS: getMultilineInput("vars"),
 	COMMANDS: getMultilineInput("command"),
+	QUIET_MODE: getBooleanInput("quiet"),
 } as const;
 
 function getNpxCmd() {
 	return process.env.RUNNER_OS === "Windows" ? "npx.cmd" : "npx";
 }
 
+function info(message: string, bypass?: boolean): void {
+	if (!config.QUIET_MODE || bypass) {
+		originalInfo(message);
+	}
+}
+
+function error(message: string): void {
+	if (!config.QUIET_MODE) {
+		originalError(message);
+	}
```

don't we need a `bypass` param here to avoid silent failures?